dbcon/ddlpackageproc/altertableprocessor.cpp

```diff
@@ -88,6 +90,14 @@ struct extentInfo
 
 namespace
 {
+
+bool checkTextTypesLengthAreUqual(const CalpontSystemCatalog::ColType& colType, const ColumnType& newType)
+{
+  auto* newTypeCharset = get_charset(newType.fCharsetNum, MYF(MY_WME));
+  auto newTypeBytesInChar = newTypeCharset ? newTypeCharset->mbmaxlen : 1;
+  return colType.colWidth * newTypeBytesInChar == newType.fLength * colType.getCharset()->mbmaxlen;
```

Comment:
If the intention to check for equality of max text len of old and new types, then why do we multiply old width by new mbMaxLen and new width by (apparently) old mbMaxLen?

This need a comment on why is that.